events/publish.go

```diff
@@ -15,23 +15,17 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-// Publish publishes events along tm buses to clients
+// Publish events using tm buses to clients. Waits on context
+// shutdown signals to exit.
 func Publish(ctx context.Context, tmbus tmclient.EventsClient, name string, bus pubsub.Bus) error {
 
 	const (
 		queuesz = 100
 	)
 	var (
-		txname  = name + "-tx"
 		blkname = name + "-blk"
 	)
 
-	txch, err := tmbus.Subscribe(ctx, txname, txQuery().String(), queuesz)
```

Comment:
I think we can nix the `txQuery()` definition.

Reply:
`deadcode` linter caught it too!